File: gui.py
#!/usr/bin/env python
# coding: utf-8

# In[ ]:


#!/usr/bin/python3


print("content-type: text/html")
print()


import subprocess as sb
import cgi

print("Welcome")
var = cgi.FieldStorage("command")
command = var.getvalue("command")
output = sb.getoutput("sudo " + command)
print(output)

File: gui.js
#!/usr/bin/env python
# coding: utf-8

# In[ ]:


function ajax(i) {
   // document.getElementById("d2").innerHTML = "Command is being Executed, Please wait..."
    var xhr = new XMLHttpRequest();


    xhr.open("GET", "http://192.168.43.140/cgi-bin/cli.py?x="+i, true);

    xhr.send();

    // Output from above url

    xhr.onload = function () {
        var output = xhr.responseText;
        document.getElementById("d2").innerHTML = output;
    }


}

function printer(id) {
    var keyword = document.getElementById(id);
    return keyword.value;
}

